python/python/psyche/vllm/run_inference_subprocess.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sys
+from pathlib import Path
+import multiprocessing
+
+os.environ["VLLM_LOGGING_LEVEL"] = "ERROR"
+
+multiprocessing.set_start_method("spawn", force=True)
+
+sys.path.insert(0, str(Path(__file__).parent.parent.parent))
+
+
+def run_inference(model_name: str, prompt: str) -> str:
+    import contextlib
+
+    from psyche.vllm.engine import UpdatableLLMEngine
+
+    original_stdout = sys.stdout
+    original_stderr = sys.stderr
+
+    with open(os.devnull, "w") as devnull:
+        sys.stdout = devnull
+        sys.stderr = devnull
+
+        try:
+            engine = UpdatableLLMEngine(
+                model_name=model_name,
+                tensor_parallel_size=1,
+                max_model_len=512,
+                gpu_memory_utilization=0.3,
+            )
+
+            sampling_params = {
+                "temperature": 0.0,  # Deterministic
+                "max_tokens": 20,
+            }
+
+            request_id = engine.add_request(prompt, sampling_params)
+            outputs = []
+            while engine.has_unfinished_requests():
+                outputs.extend(engine.step())
+
+            generated_text = outputs[0].outputs[0].text
```

Comment:
Should we check that the output is not empty? I think this will crash if that's the case